.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,36 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Issue related to manifests
+    url: https://github.com/kubeflow/manifests/issues/new
+    about: Create issue in kubeflow/manfiests repo
+  - name: Issue related to deployment
+    url: https://github.com/kubeflow/kfctl/issues/new
+    about: Create issue in kubeflow/kfctl repo
+  - name: Issue related to pipelines
+    url: https://github.com/kubeflow/pipelines/issues/new/choose
+    about: Create issue in kubeflow/pipelines repo
+  - name: Issue related to notebooks
+    url: https://github.com/kubeflow/kubeflow/issues/new/choose
```

Comment:
Will this be a self-link? I guess you need to remove the `/choose`.

Reply:
Sounds good